sycl/test/reduction/reduction_nd_lambda.cpp

```diff
@@ -0,0 +1,71 @@
+// UNSUPPORTED: cuda
+// OpenCL C 2.x alike work-group functions not yet supported by CUDA.
```

Comment:
I'd like to see a bit more clarification here. Are block reductions missing from CUDA itself or CUDA plugin does not expose them?

Reply:
Hi Alex, I updated the comment by copying it from here: https://github.com/intel/llvm/blame/sycl/sycl/test/reduction/reduction_nd_s0_dw.cpp#L2 and adding a note about intel::reduce() that is not supported yet by CUDA. 
I don't have any other details regarding CUDA support for that feature.
Thank you!

Reply:
> I don't have any other details regarding CUDA support for that feature.

It's implementable, but not in the plugin yet.  CUDA doesn't have native support for equivalents to everything in the GroupAlgorithms extension, but it did recently add support for reductions as part of its [cooperative group functionality](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#collectives-cg-reduce).

Reply:
Thank you for the link. The updated comment is explanatory enough, I think:
// Reductions use work-group builtins (e.g. intel::reduce()) not yet supported
// by CUDA.

Reply:
I completely agree.  I was just providing some more background for @alexbatashev.